core/src/com/unciv/ui/utils/ExtensionFunctions.kt

```diff
@@ -93,6 +93,16 @@ fun Actor.addBorder(size:Float, color: Color, expandCell:Boolean = false): Table
     return table
 }
 
+/** Wrap an [Actor] in a [Group] of a given size */
+fun Actor.sizeWrapped(x: Float, y: Float) = Group().also {
+    it.isTransform =
+        false // performance helper - nothing here is rotated or scaled
+    it.setSize(x, y)
+    this.setSize(x, y)
+    this.center(it)
+    it.addActor(this)
+}
```

Comment:
If you're using both `it` and `this` in the same mini lambda it gets confusing, how about a regular function where you change the actor and then wrap it in a group and return the group?

Reply:
I was going to argue for the original code, but then I got confused myself, so yeah lol. Changed.